goby.go

```diff
@@ -3,29 +3,59 @@ package main
 import (
 	"flag"
 	"fmt"
-	"github.com/pkg/profile"
 	"github.com/goby-lang/goby/bytecode"
 	"github.com/goby-lang/goby/parser"
 	"github.com/goby-lang/goby/vm"
+	"github.com/pkg/profile"
 	"io/ioutil"
+	"log"
 	"os"
 	"path"
-	"strings"
-	"log"
 	"path/filepath"
+	"strings"
 )
 
+// Version stores current Goby version
+const Version string = "0.0.1"
+
```

Comment:
No, I mean we need to document about updating this constant, create a new release, update brew formula...etc.